docs/samples/bookinfo.md

```diff
@@ -35,72 +35,18 @@ This application is polyglot, i.e., the microservices are written in different l
-
-1. Clone the istio GitHub repository and start the core Istio services (the istio-manager, the [Mixer](https://istio.io/docs/concepts/mixer.html), and the istio ingress controller).
-   
-   ```bash
-   git clone https://github.com/istio/istio
-   cd istio
-   kubectl apply -f ./kubernetes/istio-install
-   ```
-   
-   _Note: the current version of the bookinfo application MUST use the `default` Kubernetes namespace._ 
-   
-1. If you would like to view the metrics collected by Istio proxies, you need to install the [Prometheus](https://prometheus.io/) addon and start a [Grafana](https://grafana.com/) service as well.
-   
-   ```bash
-   kubectl apply -f ./kubernetes/addons/
-   ```
-
-   The Grafana image provided as part of this sample contains a built-in Istio-dashboard that you can access from:
-
-   ```
-   http://<grafana-svc-external-IP>:3000/dashboard/db/istio-dashboard
-   ```
-
-   > The addons yaml files contain services configured as type LoadBalancer. If services are deployed with type NodePort,
-   > start kubectl proxy, and edit Grafana's Istio-dashboard to use the Envoy proxy. Access Grafana via kubectl proxy:*
-
-   ```
-   http://127.0.0.1:8001/api/v1/proxy/namespaces/<ns>/services/grafana:3000/dashboard/db/istio-dashboard
-   ```
-   
-1. Install the [istioctl](../reference/istioctl.md) CLI, which provides a
-   convenient way to apply routing rules and policies for upstreams. The
-   [istio.VERSION](https://github.com/istio/istio/blob/master/istio.VERSION) file includes the download location of 
-   three OS-specific binaries: `istioctl-osx`, `istioctl-win.exe`,
-   `istioctl-linux` targeted at Mac, Windows and Linux users
-   respectively. Download the tool appropriate to your platform. For
-   example, when running istioctl on a Mac, run the following commands:
-
-   ```bash
-   source ./istio.VERSION # set ISTIOCTL env variable
-   curl ${ISTIOCTL_URL}/istioctl-osx > /usr/local/bin/istioctl
-   chmod +x /usr/local/bin/istioctl
-   ```
-
-   > Note: If you already have a previously installed version of `istioctl`, make sure that
-   > it is compatible with the manager image used in `demos/kubernetes/istio-manager.yaml`.
-   > If in doubt, download again or add the `--tag` option when running `istioctl kube-inject`.
-   > Invoke `istioctl kube-inject --help` for more details.
-
+1. Follow Istio [Installation guide](../tasks/istio-installation.md)
 {% endcapture %}
 
 {% capture discussion %}
 ## Start the Application
 
-1. Change your current working directory to the bookinfo application directory:
-
-   ```bash
-   cd demos/apps/bookinfo
-   ```
-
 1. Bring up the application containers:
 
-   ```bash
-   kubectl create -f <(istioctl kube-inject -f bookinfo.yaml)
-   ```
+```bash
+kubectl apply -f <(istioctl kube-inject -f <(\
+    curl -s https://raw.githubusercontent.com/istio/istio/master/demos/apps/bookinfo/bookinfo.yaml))
+```
```

Comment:
I changed it because some people may not have github or know how to clone our repo. And I see your point related to downloading from master. It should be fine if for alpha we will provide an archive with the files. I will change the instructions to “download and unpack the installation files(link) or clone the repository.